plugin/test/unit/helpers/config.spec.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from '../../../src/helpers/config'
 
 const SAMPLE_PROJECT_DIR = `${__dirname}/../../../../demo`
-const TEST_TIMEOUT = 60_000
+const TEST_TIMEOUT = 80_000
```

Comment:
[sand] I don't think this increase is needed at this time now that the increased timeout is being applied to the `afterEach`. Is the motivation to increase this value just to make sure we don't run into this upper limit anytime soon?

Reply:
Sounds sensible. I was being a bit too cautious for future us. I've reverted. 👍🏻